tembo-cli/tembo/docker-compose.yml.template

```diff
@@ -17,6 +17,28 @@ services:
       - "traefik.tcp.routers.{{instance.instance_name}}.entrypoints=postgresql"
       - "traefik.tcp.routers.{{instance.instance_name}}.tls.passthrough=true"
       - "traefik.tcp.services.{{instance.instance_name}}.loadbalancer.server.port=5432"
+
+  {% if instance.app_services.restapi %}
```

Comment:
This is the initial version. Need to add support for `PathPrefix` (its somehow not working) & also make things more configurable.